lib/enum_chron.rb

```diff
@@ -1,26 +1,42 @@
 # frozen_string_literal: true
 
 require "enum_chron_parser"
-require "services"
 
 # Handles automatic normalization of enum_chron fields for Holdings and HTItems
 module EnumChron
-  def initialize(params = nil)
```

Comment:
I don't know how the initialize here ever worked. Rather, we default the value of the attribute to blank, and that seems to work well enough.